.github/workflows/deploy.yml

```diff
@@ -23,12 +23,29 @@ jobs:
         with:
           python-version: 3.9.12
       - id: stop-early
-        run: if "${GITHUB_WORKSPACE}/.github/has-functional-changes.sh" ; then echo "::set-output name=status::success" ; fi
+        run: |
+          if "${GITHUB_WORKSPACE}/.github/has-functional-changes.sh" ; then 
+            echo "status=success" >> $GITHUB_OUTPUT
+          fi
 
+  check-pypi-token:
+    runs-on: ubuntu-22.04
+    outputs:
+      pypi_token_present: ${{ steps.check_token.outputs.pypi_token_present }}
+    steps:
+      - name: Check PYPI Token
+        id: check_token
+        run: |
+          if [[ -n "${{ secrets.PYPI_TOKEN }}" ]]; then
```

Comment:
> The comment you made in [the commit](https://github.com/openfisca/country-template/pull/147/commits/7afe3008235bfcbf04d30fdc9c5567eb664059a7) to explain the implementation  is really good, but it would be useful to also have it as a comment in the code to save some future smartass 😉